packages/domain/package.json

```diff
@@ -46,6 +46,7 @@
     "prettier:fix": "prettier --write .",
     "prettier": "prettier --check .",
     "test": "jest",
+    "test:watch": "jest --watch",
```

Comment:
if we agree to add that, then I'd add it to all packages so that we have the same scripts available everywhere